lib/contracts/index.ts

```diff
@@ -65,4 +66,20 @@ export const contracts: ContractDefinition[] = [
 	viewAllTransformers,
 	viewAllViews,
 	viewScheduledActions,
+
+	// Defining 'is creator of' as from `any` to `any` until we decide
+	// if we want to link from User or any other more restricted type
+	// See https://github.com/product-os/jellyfish-worker/pull/1462
```

Comment:
I think there is a brainstorm call about this today, so lets wait and see where that conversation goes before adding this in.